src/main/java/com/marklogic/spark/Options.java

```diff
@@ -21,6 +21,8 @@ public abstract class Options {
     public static final String READ_XQUERY = "spark.marklogic.read.xquery";
     public static final String READ_XQUERY_FILE = "spark.marklogic.read.xqueryFile";
     public static final String READ_VARS_PREFIX = "spark.marklogic.read.vars.";
+    public static final String READ_PARAMS_PREFIX = "spark.marklogic.read.params.";
```

Comment:
This is me trying to remember what code I wrote 6 months ago, but I believe I added this first as a way to manually define a param value in options without creating a new column / have runtime pushdown support.  From my perspective, I did this so that I could build the feature out iteratively.  From a user perspective, I don't actually know how useful this is.  It could be used to test / develop queries, or maybe to reuse a single optic query for multiple downstream use cases but have different permutations that are defined with Spark options?  You can't edit Spark options on the fly, AFAIK, so I think the value is marginal at best.